src/main/java/school/lemon/changerequest/java/introduction/hw2/Task7.java

```diff
@@ -2,6 +2,19 @@
 
 public class Task7 {
     public static int countEvens(int[] arr) {
-        return 0;
+        if (arr==null){
+            return 0;
+        }
+        int len = arr.length;
+        int number = 0;
+
+        for (int i=0;i<len;i++){
+        if (arr[i] % 2 == 0)
+            {
+                number++;
+            }
+        }
+        return number;
     }
+
```

Comment:
@IgorAfanasenko I don't really see fixes for my comment. (It's formatted now, but have you read about parity bit check ?)